src/main/java/com/openshift/jenkins/plugins/pipeline/OpenShiftServiceVerifier.java

```diff
@@ -136,7 +108,7 @@ public boolean configure(StaplerRequest req, JSONObject formData) throws FormExc
             // To persist global configuration information,
             // pull info from formData, set appropriate instance field (which should have a getter), and call save().
         	retry = formData.getInt("retry");
-        	GlobalConfig.setServiceVerifyRetry(retry);
+        	GlobalConfig.setDefaultServiceVerifyRetry(retry);
```

Comment:
Yeah, not inclined on this rename ... remember, users actually specify config at the global level (and dont' have to specify at the step level)